shap/_explanation.py

```diff
@@ -323,9 +323,10 @@ def __getitem__(self, item):
             pos += 1
 
             # skip over Ellipsis
-            if t == Ellipsis:
-                pos += len(self.shape) - len(item)
-                continue
+            if type(t) not in [np.ndarray, pd.Series]:
```

Comment:
Can you elaborate on scenarios where this change is needed? Thanks!

Reply:
oh, that was for #2325 , just showing my naive "fix" for that issue, but didn't notice it also affected this PR. Let me check what I can do!